### PR TITLE
Fix system call

### PR DESCRIPTION
**Pull request by VincentCordobes**

### CODE REVIEW

File: autoload/phonetics/audio/common.vim [hunk 1]
```diff
@@ -11,12 +11,6 @@ endf
...
 fun! phonetics#audio#common#Scrap(regex, word)
   let l:html = phonetics#core#GetHTML(a:word)
-
-  try
-    let [l:match, l:transcription, l:audio_url; l:rest] = matchlist(l:html, a:regex)
-    retu [l:transcription, l:audio_url]
-  cat
-    retu []
-  endt
+  call system('mpv ' . shellescape(l:audio_url))
```
Comment (soywod) [inline autoload/phonetics/audio/common.vim]: You don't think it's good to keep both transcription and audio ?

File: autoload/phonetics/core.vim [hunk 2]
```diff
@@ -1,6 +1,7 @@
 fun! phonetics#core#GetHTML(word)
-  let l:url = 'curl -L https://www.oxfordlearnersdictionaries.com/definition/english/' . a:word . '_1?q=' . a:word
-  retu system(l:url)
+  let l:url = 'https://www.oxfordlearnersdictionaries.com/definition/english/' . a:word . '_1?q=' . a:word
+  let l:cmd = 'curl -L ' . shellescape(l:url)
```
Comment (soywod) [inline autoload/phonetics/core.vim:3]: :+1:

File: autoload/phonetics/audio/common.vim [hunk 3]
```diff
@@ -11,12 +11,7 @@ endf
...
 fun! phonetics#audio#common#Scrap(regex, word)
   let l:html = phonetics#core#GetHTML(a:word)
-
-  try
-    let [l:match, l:transcription, l:audio_url; l:rest] = matchlist(l:html, a:regex)
-    retu [l:transcription, l:audio_url]
-  cat
-    retu []
-  endt
+  let [l:match, l:transcription, l:audio_url; l:rest] = matchlist(l:html, a:regex)
```
Comment (VincentCordobes) [inline autoload/phonetics/audio/common.vim:14]: better to have an error than ignoring it, isn't it ?

Comment (soywod) [reply autoload/phonetics/audio/common.vim:14]: it's caught in the upper function

**Pull request merged**